### PR TITLE
PIM-5207 : split and rename features for DCI

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                 | A
| ----------------- | ---
| Added Specs       |n
| Added Behats      |only splitted features
| Travis CI is ok   |y
| Changelog updated |y

### CODE REVIEW

**Comment by willy-ahva** [comment]
GTM

**Comment by jmleroux** [comment]
no idea how to rename `features/mass-action/edit_common_attributes-01.feature` and `features/mass-action/edit_common_attributes-02.feature`

**Comment by jjanvier** [comment]
gtm

File: CHANGELOG-1.5.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 - PIM-4743: Added the possibility to use optgroup in Oro ChoiceFilter
 - PIM-4347: `Pim\Bundle\CatalogBundle\Repository\ProductRepositoryInterface` now extends `Doctrine\Common\Persistence\ObjectRepository`
 - PIM-5067: Change the JUnit formatter of behats logs
+- PIM-5207: split and rename features for DCI
```
Comment (jjanvier) [inline CHANGELOG-1.5.md]: I would remove
(no one knows what is DCI, this is just internal stuff)

Comment (jmleroux) [inline CHANGELOG-1.5.md]: agree

**Pull request merged**